sdk/core/azure-core/test/ut/transport_adapter_base_test.cpp

```diff
@@ -268,8 +268,11 @@ namespace Azure { namespace Core { namespace Test {
     auto response = m_pipeline->Send(request, Azure::Core::Context::ApplicationContext);
     checkResponseCode(response->GetStatusCode());
 
-    auto expectedResponseBodySize = std::stoull(response->GetHeaders().at("content-length"));
-    CheckBodyFromStream(*response, expectedResponseBodySize);
+    if (response->GetStatusCode() == Http::HttpStatusCode::Ok)
```

Comment:
Doesn't `checkResponseCode` above already validate this to be true?

Reply:
It does check the value. But the test doesn't stop running after a failure, so it plows on to line 272 and crashes in that code because the response body isn't set.